### ISSUE TITLE
Allow users to manually delete their accounts and add "Pending deletion" account state

### ISSUE THREAD

**Issue by mesosoi** [mesosoi/silverwaregames-io-issue-tracker#39]
Allow users to manually delete their accounts and add "Pending deletion" account state

**Comment by michaelplzno** [comment]
I think there is a law in the EU that requires someone who deletes their account to be completely "forgotten" that is, their data dies forever. I assume when a user deletes their account that happens right?

**Comment by mesosoi** [comment]
> I think there is a law in the EU that requires someone who deletes their account to be completely "forgotten" that is, their data dies forever. I assume when a user deletes their account that happens right?

Yes there is. And no, it does not happen yet. As for now if someone will request us to delete their account we'll have to do this manually.
